README.md

```diff
@@ -64,6 +64,21 @@ At boot time, you might need `adb shell setenforce 0` for B2G to boot (flash_sar
   
 If need to output a zip ROM file, you can use `./build.sh dist DIST_DIR=dist_output` instead of `./build.sh` in step 5.
 
+## Generic B2G System Images (GSI)
+
+1. Fetch the code: `REPO_INIT_FLAGS="--depth=1" ./config.sh gsi_b2g`
+2. Setup your environment to fetch the custom NDK: `export LOCAL_NDK_BASE_URL='ftp://ftp.kaiostech.com/ndk/android-ndk'`
+3. Install Gecko dependencies: `cd gecko && ./mach bootstrap`, choose option 4 (Android Geckoview).
+4. Apply patch: `./patcher/patcher.sh`
```

Comment:
This is not here?

Reply:
need it 
https://github.com/ittat/patcher/tree/b2g-gsi
https://github.com/ittat/manifests/blob/b2g-gsi/gsi.xml#L16

Reply:
ok, it's not `+x`

Reply:
patching failure:
```
Application de  Add route to gateway on null netmask
error: le patch a échoué : src/java/com/android/internal/telephony/dataconnection/DataConnection.java:25
error: src/java/com/android/internal/telephony/dataconnection/DataConnection.java : le patch ne s'applique pas
le patch a échoué à 0001 Add route to gateway on null netmask
astuce: Utilisez 'git am --show-current-patch=diff' pour visualiser le patch en échec
```

Reply:
quite a bad day ~
We don’t actually need this patch...